spec/requests/all_casa_admins/dashboard_spec.rb

```diff
@@ -17,7 +17,8 @@
     it { is_expected.to have_http_status(:success) }
 
     it "shows casa orgs" do
-      page = request.parsed_body
+      # Code changes to fix response as earlier HTML String instead of Nokogiri::HTML5::Document object as received in Rails 7.1.0 to pass expectation
+      page = request.parsed_body.to_html
```

Comment:
Had to update code as request.parsed_body in Rails 7.1 was returning  Nokogiri::HTML5::Document object which was failing spec

Reply:
makes sense